roles/docker-storage-setup/tasks/main.yaml

```diff
@@ -41,6 +46,7 @@
 
 - name: Install Docker
   package: name=docker state=present
+  when: ansible_pkg_mgr != "unknown"
```

Comment:
https://github.com/openshift/openshift-ansible-contrib/blob/master/roles/prerequisites/tasks/main.yaml

Check this. When using the library it will generate the facts to automatically know if a host is atomic or not

Reply:
Ah cool. I'd assumed you had to go into the BYO playbook before it would set that var. Will update...